src/components/Inventory/withdraw/flow/SelectBatchesWithdrawnQuantity.tsx

```diff
@@ -178,21 +183,31 @@ export default function SelectBatches(props: SelectBatchesWithdrawnQuantityProps
         let readyQuantityWithdrawnError = '';
         let notReadyQuantityWithdrawnError = '';
         if (rec.readyQuantityWithdrawn) {
-          if (+rec.readyQuantityWithdrawn > +rec.readyQuantity) {
-            readyQuantityWithdrawnError = strings.WITHDRAWN_QUANTITY_ERROR;
+          if (isNaN(rec.readyQuantityWithdrawn) || +rec.readyQuantityWithdrawn < 0) {
+            readyQuantityWithdrawnError = strings.INVALID_VALUE;
             noErrors = false;
           } else {
-            readyQuantityWithdrawnError = '';
+            if (+rec.readyQuantityWithdrawn > +rec.readyQuantity) {
+              readyQuantityWithdrawnError = strings.WITHDRAWN_QUANTITY_ERROR;
+              noErrors = false;
+            } else {
+              readyQuantityWithdrawnError = '';
+            }
           }
         } else {
           unsetValues++;
         }
         if (rec.notReadyQuantityWithdrawn) {
-          if (+rec.notReadyQuantityWithdrawn > rec.notReadyQuantity) {
-            notReadyQuantityWithdrawnError = strings.WITHDRAWN_QUANTITY_ERROR;
+          if (isNaN(rec.notReadyQuantityWithdrawn) || +rec.notReadyQuantityWithdrawn < 0) {
```

Comment:
could extract this into a function maybe

const isInvalidQuantity = (val: any) => isNaN(val) || (+val < 0);